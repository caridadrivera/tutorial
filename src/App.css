.App {
 width: 100%;
 height: 50%;
}


.App-link {
  color: #61dafb;
}

