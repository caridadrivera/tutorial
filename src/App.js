import React from 'react';
import './App.css';
import { useHistory } from 'react-router-dom';
import { BrowserRouter as Router, Route, Switch,Link } from 'react-router-dom';
// import AutoButton from './Components/autoBtn';
// import FireButton from './Components/fireBtn';
// import LifeButton from './Components/lifeBtn';
import MainPage from './Components/mainPage';
import MainAutoPage from './Components/mainAutoPage';
import MainFirePage from './Components/mainFirePage';
import MainLifePage from './Components/mainLifePage';

export default function App() {
 
  
  
  return (
    
      <Router>
          <Switch>
            <Route exact path="/" component={MainPage}></Route>
          </Switch>
        
          <Switch>
            <Route path="/auto" component={MainAutoPage}></Route>
          </Switch>
          
      </Router>
        
    
  );
}
