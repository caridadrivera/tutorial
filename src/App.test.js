import React from 'react';
import { render } from '@testing-library/react';
import App from './App';
import MainPage from './components/mainPage';
import MainAutoPage from './components/mainAutoPage';

import renderer from 'react-test-renderer';

// test('renders learn react link', () => {
//   const { getByText } = render(<App />);
//   const linkElement = getByText(/learn react/i);
//   expect(linkElement).toBeInTheDocument();
// });


it('renders correctly', () =>{
  const tree = renderer.create(<MainPage/>).toJSON();
  expect(tree).toMatchSnapshot()
})


it('it renders the main auto page', () => {
  const autoPage = renderer.create(<MainAutoPage/>).toJSON();
  expect(autoPage).toMatchSnapshot()
})

