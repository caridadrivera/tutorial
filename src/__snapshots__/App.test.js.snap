// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders the main auto page 1`] = `
<div
  className="autoPage"
>
  <button
    className="autoQuoteBtn"
  >
    Quoting
  </button>
  <button
    className="autoBillingBtn"
  >
    Billing
  </button>
  <button
    className="autoAccountBtn"
  >
    Account Management
  </button>
</div>
`;

exports[`renders correctly 1`] = `
<div
  className="mainButtonContainer"
>
  <div
    className="autoButton"
  >
    <a
      href="/auto"
      onClick={[Function]}
    >
      AUTO
    </a>
  </div>
  <div
    className="fireButton"
  >
    <a
      href="/fire"
      onClick={[Function]}
    >
      FIRE
    </a>
  </div>
  <div
    className="lifeButton"
  >
    <a
      href="/life"
      onClick={[Function]}
    >
      LIFE
    </a>
  </div>
</div>
`;
